src/views/Product/View.tsx

```diff
@@ -44,7 +44,7 @@ const extractMeta = (product: ProductDetails_product) => ({
     },
   ],
   description: product.seoDescription || product.descriptionJson,
-  image: maybe(() => product.thumbnail.url, null),
+  image: product.thumbnail.url || null,
```

Comment:
`maybe` suggests that we may not have all needed data for some time

```suggestion
  image: product?.thumbnail?.url || null,
```